apps/admin-gui/src/app/users/pages/user-detail-page/user-dashboard/dashboard-card/dashboard-card.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="card p-3">
-  <div class="flex-column card-title">
-    <mat-icon [svgIcon]="svgIcon" class="cover-image"></mat-icon>
+  <div class="d-flex flex-row align-items-center center d-grid gap-2 card-title">
```

Comment:
This should suffice. Also please squash the commits.
```suggestion
  <div class="d-flex flex-row align-items-center gap-2 card-title">
```

Reply:
Squashed and fixed.